eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -268,3 +268,28 @@ jobs:
         inputs:
           mergeTestResults: true
           testRunTitle: '$(OSName) on Java $(JavaVersion)'
+
+      - task: PublishTestResults@2
```

Comment:
With the latest changes to JaCoCo configuration, we need one additional step - `Generate aggregate code coverage report` that's added in `client.yml`. We don't need the coverage report generated for each PR, so, when you add this step, include a condition to run it only in a non-PR build.